models/cow_protocol/ethereum/cow_protocol_app_data.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(alias='app_data',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "cow_protocol",
+                                    \'["bh2smith"]\') }}'
+)}}
```

Comment:
I don't actually know if this is the correct configuration to ensure the "view" is updated whenever the reference table is.

Reply:
this is correct config for a view